services/app-web/src/common-code/data-helpers/makeDocumentKeyLookupList.test.ts

```diff
@@ -0,0 +1,44 @@
+import { makeDocumentList } from './makeDocumentKeyLookupList'
```

Comment:
In my refactor I moved Maz’s document table code out of /common-code/data-helpers and just put it in pages/SubmissionSummary directly. 

This for sure doesn’t belon inside of /common-code b/c I’m trying to only put code in there that is imported by the front end and the back end, and this code operates on the GQL types so it wouldn’t make sense on the backend anyway. 

Maybe something like “documentsHelpers” could be a package in /src that could have your code and Maz’s in it?